### PR TITLE
Update the System Requirement of Amazon Linux in Infra Server

### PR DESCRIPTION
**Pull request by dishanktiwari2501**
## Description

Amazon Linux 2 is a supported platform that we test and it needs to be included on this page.

## Related PRs

- https://github.com/chef/chef-web-docs/issues/3376

### CODE REVIEW

File: themes/docs-new/layouts/shortcodes/adopted_platforms_server.md [hunk 1]
```diff
@@ -39,5 +39,10 @@ The following table lists the commercially-supported platforms and versions for
 <td><code>x86_64</code></td>
 <td><code>16.04</code>, <code>18.04</code>, <code>20.04</code></td>
 </tr>
+<tr>
+<td>Amazon Linux 2</td>
```
Comment (kagarmoe) [inline themes/docs-new/layouts/shortcodes/adopted_platforms_server.md]: We should list this in alphabetical order.

**Review by kagarmoe — CHANGES_REQUESTED** [review]
Thanks! Could you put the list into alphabetical order?

File: themes/docs-new/layouts/shortcodes/adopted_platforms_server.md (hunk 1 above)
Comment (dkumaras) [reply themes/docs-new/layouts/shortcodes/adopted_platforms_server.md]: done

**Comment by dkumaras** [comment]
I thought I need to do, and just did a small change - @kagarmoe & @dishanktiwari2501

**Review by kagarmoe — APPROVED** [review]

**Review by kagarmoe — APPROVED** [review]

**Review by kagarmoe — APPROVED** [review]

**Pull request merged**